src/Options.cpp

```diff
@@ -56,6 +56,8 @@ bool ReadIni(HealTableOptions& pOptions)
 		pOptions.Windows[i].ExcludeOffSquad = healtable_ini.GetBoolValue(section, "exclude_off_squad", pOptions.Windows[i].ExcludeOffSquad);
 		pOptions.Windows[i].ExcludeMinions = healtable_ini.GetBoolValue(section, "exclude_minions", pOptions.Windows[i].ExcludeMinions);
 		pOptions.Windows[i].ExcludeUnmapped = healtable_ini.GetBoolValue(section, "exclude_unmapped", pOptions.Windows[i].ExcludeUnmapped);
+		pOptions.Windows[i].ExcludeHealing = healtable_ini.GetBoolValue(section, "exclude_healing", pOptions.Windows[i].ExcludeHealing);
+		pOptions.Windows[i].ExcludeBarrierGeneration = healtable_ini.GetBoolValue(section, "exclude_barrier_generation", pOptions.Windows[i].ExcludeBarrierGeneration);
```

Comment:
The ini code is just there because of conversion from old format - don't need to add new fields in there

Reply:
Code removed